### DISCUSSION TITLE
Show off your glance

### DISCUSSION THREAD

**Discussion by 80imike** [glanceapp/glance#102] [General]
"Glance" is a great project, come here and show off your customized "Glance"!

This is my: https://bestgeek.org/

![screenshot-geek11](https://github.com/glanceapp/glance/assets/1665201/c7db2048-bc44-43c8-81f5-7fb8b0f00625)

-  update 2024.07.02
![image](https://github.com/glanceapp/glance/assets/1665201/54131a1d-4145-470d-8b56-1f0e88e66a41)

**Comment by Famku** [comment]
![screencapture-start-famku-net-2024-06-03-06_28_39](https://github.com/glanceapp/glance/assets/27890413/d4d366bd-c995-4d33-be1d-23c540df4639)

**Reply by vishalkadam47** [reply]
How did you add Analogue Clock

**Reply by Famku** [reply]
it's from codepen 
https://codepen.io

**Reply by vishalkadam47** [reply]
ok but how do i add it to my dashboard is there a way using `glance.yml` or manually added in the code, please guide

**Reply by Famku** [reply]
it is just an iframe

**Reply by vishalkadam47** [reply]
ok got it thanks

**Reply by derencio** [reply]
Newbie question here: I've tried many ways to add it but I keep getting it in a frame which doesn't look cool. Would you mind sharing the actual string to get that running as you did?

**Reply by zapovit** [reply]
@derencio not clear what you mean by frame in this context. Here is a snippet I use to get a pretty seamless iframe, but I think `frameless` is a global widget option that you can use to remove the "frame" around each widget (1px border, shadow, and background colour). There is also the `iframe` which might receive a `frameborder` from the browser in certain situations. Then there are a number of hacks to get transparent backgrounds in the `iframe` but they depend on you controlling the page within (so the codepen thing wouldn't necessarily have a transparent background unless you host the file on your own server).

Simple html example
```
- type: html
  frameless: true
  source: |
    <iframe src="../html/watchlist.html" id="watchList" width="100%" height="154px" frameborder="0"></iframe>
```
Then apply the css style in your custom css (follow the docs to setup custom css)
```
iframe#watchList {
        border:none;
        overflow:hidden;
        background:none;
        background-color:transparent;
        color-scheme:normal;
}
```
This css makes sure: no border, no scrollbar in x or y direction, no background colour, and a reset `color-scheme` which seems to help with detecting light/dark mode user preference within the frame.

This gives a bit more control than the normal iframe widget which should be frameless by default if I'm not mistaken. You typically need to determine the height of the page in the frame that you want to show and enforce this on the iframe itself.

**Comment by Hecbert** [comment]
![image](https://github.com/glanceapp/glance/assets/98398306/3abcee6c-2fef-40fd-9c40-6c157406d604)

**Comment by zapovit** [comment]
here is a video. based on the search example. It uses custom html for the main search bar/bangs popover. The tradingview stock widget is actually an iframe with a heavily styled web-component. JustWatch widget is another custom thing using OpenRSS, FreshRSS, and wsrv.nl. Photos is a custom php extension that gets a random page of results from Pexels to cycle through for the day. 

<img width="1920" height="964" alt="Screenshot from 2026-06-19 15-15-39" src="https://github.com/user-attachments/assets/fed7f567-8968-4d88-b469-46de59a3dea4" />

[Screencast from 2026-06-19 15-02-46.webm](https://github.com/user-attachments/assets/2017fcd0-7976-41b1-8909-95e4234bb36f)

**self-hosting dashboard:**
<img width="1920" height="964" alt="Screenshot from 2026-06-19 15-18-56" src="https://github.com/user-attachments/assets/65518e06-c129-4a98-842a-884a1933425f" />
<img width="1920" height="964" alt="Screenshot from 2026-06-19 15-19-12" src="https://github.com/user-attachments/assets/bfefb4f9-81f1-4598-8c86-15c7360bf3f9" />

**Reply by zapovit** [reply]
<img width="1226" height="866" alt="image" src="https://github.com/user-attachments/assets/354ef452-0551-4845-b579-3880bae060ab" />
custom masonry columns

**Reply by zapovit** [reply]
Prototyping (unassisted) a PHP RSS parser extension which optionally integrates the vibe-coded gReader backend I've posted elsewhere in this and the FreshRSS repo.
<img width="1080" height="509" alt="Screenshot_20260711_183446_Fennec" src="https://github.com/user-attachments/assets/c3a8933a-e55a-4b4d-9946-b3785de848f7" />
